CMakeLists.txt

```diff
@@ -7,6 +7,14 @@ ELSE(TAOS_CI_INSTALL_DIR)
     SET(TAOS_CI_INSTALL_DIR "/var/www/html/TAOS-CI")
 ENDIF(TAOS_CI_INSTALL_DIR)
 
+IF (NOT CMAKE_INSTALL_SYSCONFDIR)
+  set(CMAKE_INSTALL_SYSCONFDIR "etc" CACHE PATH "Read-only single-machine data (etc)")
+ENDIF()
+
+IF (NOT CMAKE_INSTALL_LOCALSTATEDIR)
+  set(CMAKE_INSTALL_LOCALSTATEDIR "var" CACHE PATH "Modifiable single-machine data (var)")
+ENDIF()
```

Comment:
ENDIF(NOT CMAKE_INSTALL_LOCALSTATEDIR) is looks better.

Reply:
updated.